docker-compose.yml

```diff
@@ -12,10 +12,10 @@ services:
     ports:
       - "8080:8080"
     environment:
-      JDBC_URL: "jdbc:postgresql://db:5432/db"
-      JDBC_USERNAME: "postgres"
-      JDBC_PASSWORD: "postgres"
-    command: ["./wait-for-it.sh", "db:5432", "--timeout=300", "--" , "java", "-jar", "/usr/src/app/CQLStorage.jar"]
```

Comment:
this is "in progress" because I'd like to confirm this approach with the GT dev team

I'm not sure if we need to remove this `./wait-for-it.sh` script entirely or just have it point to our RDS instance

this could potentially be another instance of variable substitution